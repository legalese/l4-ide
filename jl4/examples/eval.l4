x MEANS 2 + 2
y MEANS FALSE

#CHECK x * 8
#EVAL x * 8
#EVAL IF y THEN 3 ELSE 5
