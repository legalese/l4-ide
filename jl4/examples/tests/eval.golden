Parsing successful
eval.l4:22:8-13:
NUMBER
